### PR TITLE
add cob4-16 uh

### PR DESCRIPTION
**Pull request by fmessmer**
add support for cob4-16 (Hertfordshire)

### CODE REVIEW

File: cob4-16/calibration/calibration_offset.urdf.xacro [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+	<xacro:property name="offset_gripper_left_yaw" value="0.0"/>
+
+	<!-- *********************************** -->
+	<!-- kinematic chain reference positions -->
+	<!-- *********************************** -->
+	<!-- this feature is no longer supported as urdfdom does not parse the according 'calibration_rising' tag -->
+
+	<!-- cam3d left | handeye calibration | relative to torso_3_link -->
+	<xacro:property name="offset_torso_cam3d_left_x" value="0.0"/>
+	<xacro:property name="offset_torso_cam3d_left_y" value="0.025"/>
```
Comment (floweisshardt) [inline cob4-16/calibration/calibration_offset.urdf.xacro]: should be all "0" if not calibrated

File: cob4-16/calibration/calibration_offset.urdf.xacro [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+	<xacro:property name="offset_torso_cam3d_left_z" value="0.0"/>
+	<xacro:property name="offset_torso_cam3d_left_roll" value="0.0"/>
+	<xacro:property name="offset_torso_cam3d_left_pitch" value="0.0"/>
+	<xacro:property name="offset_torso_cam3d_left_yaw" value="0.05"/>
```
Comment (floweisshardt) [inline cob4-16/calibration/calibration_offset.urdf.xacro]: should be all "0" if not calibrated

File: cob4-16/calibration/calibration_offset.urdf.xacro [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+
+	<!-- cam3d right | handeye calibration | relative to torso_3_link -->
+	<xacro:property name="offset_torso_cam3d_right_x" value="0.0"/>
+	<xacro:property name="offset_torso_cam3d_right_y" value="0.0"/>
+	<xacro:property name="offset_torso_cam3d_right_z" value="0.0"/>
+	<xacro:property name="offset_torso_cam3d_right_roll" value="0.0"/>
+	<xacro:property name="offset_torso_cam3d_right_pitch" value="0.0"/>
+	<xacro:property name="offset_torso_cam3d_right_yaw" value="0.05"/>
```
Comment (floweisshardt) [inline cob4-16/calibration/calibration_offset.urdf.xacro]: should be all "0" if not calibrated

**Review by floweisshardt — DISMISSED** [review]

**Comment by fmessmer** [comment]
@ipa-fmw please confirm again...travis will fail until ipa320/cob_robots#733 is merged

**Review by floweisshardt — APPROVED** [review]
looks good

**Pull request merged**